modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -172,8 +175,11 @@ The OTLP receiver ingests data using the OpenTelemetry protocol (OTLP).
           receivers: [otlp]
 ----
 <1> The OTLP gRPC endpoint. If omitted, the default `+0.0.0.0:4317+` is used.
-<2> The OTLP HTTP endpoint. If omitted, the default `+0.0.0.0:4318+` is used.
-<3> The TLS server side configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<2> The TLS server side configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<3> Path to the TLS certificate to use by the server to verify a client certificate. (optional) This sets the ClientCAs and ClientAuth to RequireAndVerifyClientCert in the TLSConfig. Please refer to link:https://godoc.org/crypto/tls#Config[Golang TLS Config] for more information.
```

Comment:
```suggestion
<3> The path to the TLS certificate at which the server verifies a client certificate. Optional: You can set the `ClientCAs` and `ClientAuth` to `RequireAndVerifyClientCert` in the `TLSConfig`. See link:https://godoc.org/crypto/tls#Config[Golang TLS Config] for more information.
```

Reply:
I wanted to say that this setting is optional